docs/administration/configuration/configuring-user-quotas/index.md

```diff
@@ -0,0 +1,65 @@
+it is possible to configure quotas to limit the usage of resources, such as CPU and disk.
+This page describes how to enable quotas on your REANA cluster and how to set usage limits.
+If you are looking for documentation from the user perspective, please check [this page](https://docs.reana.io/advanced-usage/user-quotas/).
+
+## Enabling and disabling quotas
+
+Quotas are enabled by default in REANA, but you can set the [`quota.enabled`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value to `false` to disable them.
+
+REANA automatically creates two quota resources for you:
+
+- `cpu` to limit the amount of CPU time available to run workflows;
+- `disk` to limit the storage space available for file uploads.
+
+In addition to enabling quotas, you might want to decide how frequently quotas should be updated.
+This is done using two options: [`quota.periodic_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) and [`quota.workflow_termination_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana).
+
+The [`quota.periodic_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value defines when a periodic cron job should be scheduled to update quotas. This is specified using a [cron expression](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) and by default it is set to `"0 3 * * *"`, which means that it is scheduled to run once every night at 3AM.
+
+The [`quota.workflow_termination_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value is responsible for defining which resources should be updated after the workflow is in its final state (finished, failed, stopped, deleted). By default, quotas are not updated at the end of the execution of a workflow.
+
+Below you can find an example configuration:
+
+```yaml
+quota:
+  enabled: true
+  periodic_update_policy: "0 3 * * *"  # everyday at 3AM
+  workflow_termination_update_policy: "cpu,disk"  # after workflow finishes or fails update its cpu and disk resources usage
+```
+
+## Setting default quota limits
+
+When user quotas are enabled, no limit on the usage of resources is enforced by default. You can set a default quota limit that will apply to all new users with the [`quota.default_disk_limit`](https://github.com/reanahub/reana/tree/master/helm/reana) and [`quota.default_cpu_limit`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm values:
+
+```yaml
+quota:
+  default_disk_limit: 5000000000  # bytes (4768 MiB) 
+  default_cpu_limit: 3600000      # milliseconds (1 hour)
+  ...
+```
+
+!!! warning
+    If the default quota values are changed, they will only be applied to newly created users. Users that already exist will not be affected by this change.
+
+## Setting quotas for selected users
+
+In certain situations, you might want to set different quotas for specific users.
```

Comment:
In addition to setting the default quota limits for all users, you may want to set different quota usage limits for different specific groups of users.